voorhoede/head-start
# feature: directly link content records (routing)

# Changes

Directly directly link content records instead of using a generic InternalLink component for better content editing experience. See new decision log entry for detailed motivation.

# Associated issue

Resolves #160 

# How to test

1. Open preview link
2. Tap on the link for documentation
4. Verify that this links to a valid oage
5. Verify in Dato (environment 160-link-records) that this link is a Record Reference

# Checklist
